index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Medini Website Design</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <section class="container">
+        <a><h3 class="main_heading">My Logo</h3></a>
+        <nav>
+            <img class="menubutton" src='./images/Union.svg' alt="menubutton">
+        </nav>
+    </section>
+    <section class="container mobilepara">
+        <h1>I'm Medini<br>Glad to see<br>you!</h1>
+        <p>I'm a software developer! I can help you
+            build a product, feature or website Look
+            through some of my work and 
+            experience! If you like what you see and
+            have a project you need coded, don't
+            hesitate to contact me.</p>
+        <h4>LET'S CONNECT</h4>
+        <!--img src='./images/social_media_icons.svg' class="social_media_icons" alt="social media icons"!>!-->
+        <ul class="social_media_icons">
+            <li><a href=#><img src='./images/twitter.svg' alt="twitter icon"></a></li>
+            <li><a href=#><img src='./images/Linkedin_icon.svg' alt="LinkedIn icon"></a></li>
+            <li><a href=#><img src='./images/m_icon.svg' alt="M icon"></a></li>
+            <li><a href=#><img src='./images/cat_icon.svg' alt="Cat icon"></a></li>
+            <li><a href=#><img src='./images/hand_icon.svg' alt="Hand icon"></a></li>
+        </ul>
+    </section>
+    <div class="grid-container">
+        <div class="grid-item item1">
+            <img src='./images/portfolio1.svg' alt="portfolio image">
+            <h1>Tonic</h1>
+            <div>
+                <h4>CANOPY</h4>
+                <ul>
+                    <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
+                <li><h4>css</h4></li>
+                <li><h4>javaScript</h4></li>
```

Comment:
- Instead of ``h4`` should be an ``a`` tag.

Reply:
Changed it to "a" tag